doc/classes/String.xml

```diff
@@ -920,66 +920,93 @@
 			<return type="bool" />
 			<param index="0" name="right" type="String" />
 			<description>
+				Returns [code]true[/code] if both strings do not contain the same sequence of characters.
 			</description>
 		</operator>
 		<operator name="operator !=">
 			<return type="bool" />
 			<param index="0" name="right" type="StringName" />
 			<description>
+				Returns [code]true[/code] if this [String] is not equivalent to the given [StringName].
 			</description>
 		</operator>
 		<operator name="operator %">
 			<return type="String" />
 			<param index="0" name="right" type="Variant" />
 			<description>
+				Formats the [String], replacing the placeholders with one or more parameters.
+				To pass multiple parameters, [param right] needs to be an [Array].
+				[codeblocks]
+				[gdscript]
+				print("I caught %d fishes!" % 2) # Prints "I caught 2 fishes!"
+
+				var my_message = "Travelling to %s, at %2.2f per second."
+				var location = "Deep Valley"
+				var speed = 40.3485
+				print(my_message % [location, speed]) # Prints "Travelling to Deep Valley, at 40.35 km/h."
+				[/gdscript]
+				[csharp]
+				// In C#, there is no direct equivalent to this operator.
+				// See the String.format() method, instead.
```

Comment:
Should be extracted into the description, otherwise this is not translatable and not useful to international readers.